src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/index.js

```diff
@@ -54,9 +55,18 @@ Component.register('sas-blog-detail', {
         },
         'blog.title': function (value) {
             if (typeof value !== 'undefined') {
-                this.blog.slug = slugify(value, {
-                    lower: true
-                });
+                const criteria = new Criteria();
+                criteria.addFilter(
+                    Criteria.equals('slug', value)
+                );
+                this.repository.search(criteria, Shopware.Context.api)
+                    .then(result => {
+                        if (this.$route.name==="blog.module.detail"){
+                            this.slugDetailsPage(result,value);
+                        }else {
+                            this.slugCreatePage(result,value)
+                        }
+                    });
```

Comment:
Please use space ";" and "," reasonable
```suggestion
                this.repository.search(criteria, Shopware.Context.api).then(result => {
                        if (this.$route.name === "blog.module.detail") {
                            this.slugDetailsPage(result, value);
                        } else {
                            this.slugCreatePage(result, value);
                        }
                    });
```